src/DotNetEnv/Extensions/LinqExtensions.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DotNetEnv.Extensions
+{
+    public static class LinqExtensions
+    {
+        public static Dictionary<string, string> ToDotEnvDictionary(this IEnumerable<KeyValuePair<string, string>> @this, CreateDictionaryOption option = CreateDictionaryOption.TakeLast)
```

Comment:
I don't know this syntax of the extra `@this` -- what's that for?

Reply:
that's just my preference for using @this as `this`-variable in extensions... you can name it anything different of course.

The leading @ is because you cannot name a variable `this`, because that's a different thing of course...

Reply:
But it is already named `this`... We can leave it as that (same as it was before in the previous extension?

Reply:
marking it as this (because otherwise it wouldn't be an extension method) and naming it `@this` are two different things I guess ;)

But no problem, I rename it to `kvps` again.

The "default" naming if we stick to Linq-namings would be `source` (most of the cases I think, there are some different if I remember correctly). But in the end it's just a name of course...

I still do prefer the `@this`, because that directly makes me aware when reading inside the method (not the signature) that the variable I'm working on is the "self-reference" for a extension-method.
It would be quite uncommon that anyone uses `@this` in a different context, while `source` or any different naming may occurs in different contexts too, possibly even in extension-methods but not as the "this"-marked parameter.